scss/styles.scss

```diff
@@ -0,0 +1,6 @@
+body {
+  background-color: $wild-sand;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
We don't use tag elements to set CSS rules. Use `.body` class intead.

Reply:
Done ✅